package.json

```diff
@@ -89,8 +89,8 @@
     "express": "4.17.1",
     "graphiql": "^1.4.1",
     "graphiql-subscriptions-fetcher": "0.0.2",
-    "graphql": "15.4.0",
-    "graphql-ws": "4.1.2",
+    "graphql": "15.5.0",
+    "graphql-ws": "4.1.3",
```

Comment:
Regarding locking to `graphql-ws@4.1.3`, we could bypass the `ERR_PACKAGE_PATH_NOT_EXPORTED` by not using `require.resolve`, but instead do a plain path concat:

```diff
- const filePath = require.resolve(npmPath);
+ const filePath = path.join(process.cwd(), 'node_modules', npmPath);
```

WDYT?